datacenter/dtr/2.2/guides/admin/configure/external-storage/index.md

```diff
@@ -6,116 +6,72 @@ title: Configure DTR image storage
 ---
 
 After installing Docker Trusted Registry, one of your first tasks is to
-designate and configure the Trusted Registry storage backend. This document provides the following:
+designate and configure the Trusted Registry storage backend.  This document
+provides the following:
 
 * Information describing your storage backend options.
 * Configuration steps using either the Trusted Registry UI or a YAML file.
 
-While there is a default storage backend, `filesystem`, the Trusted Registry offers other options that are cloud-based. This flexibility to configure to a different storage backend allows you to:
+The default storage backend, `filesystem`, stores and serves images from the
```

Comment:
In your head it's very clear what `in a GA setup this fails` means, but from interactions with users, this is totally not obvious.
Maybe say something like

When your DTR deployment has a single replica, then you can use the filesystem on that node to store Docker images. If your DTR deployment has multiple replicas, you need to ensure all the replicas use the same storage backend, so that you can pull images independently of the replica that gets your request.